README.md

```diff
@@ -101,7 +101,14 @@ If you need more customization, you can write a module:
 
 There are a few of types of modules:
 
-
+- Server - Use to set properties of the Bucky Server
```

Comment:
I'll take another look at this, I believe I was just trying to specify the config options, I may have missed the heading about modules.